user-console/src/main/java/org/pentaho/mantle/client/solutionbrowser/SolutionBrowserPanel.java

```diff
@@ -413,6 +413,10 @@ public boolean isSupportedExtension( String extension ) {
     return supportedFileExtensions.contains( extension );
   }
 
+  public static List<String> getSupportedGenericFileExtensions() {
```

Comment:
Does not appear related to these stories.

Reply:
I will fix this along with https://github.com/pentaho/pentaho-platform/pull/5638#discussion_r1644574159 and do it in my upcoming PR with UTs and other small changes (including VFS File supported file extension on the server side)